### PR TITLE
Changed text in 2.7 XPaths in XProc

### PR DESCRIPTION
**Pull request by xml-project**
to XPath 3.1 static and dynamic context items

### CODE REVIEW

**Comment by ndw** [comment]
If you restart the Travis build, does it work?

File: langspec/xproc11/xproc.xml [hunk 1]
```diff
@@ -1027,7 +1030,7 @@ refer to that variable will raise an error.</para>
               </listitem>
             </varlistentry>
             <varlistentry>
-              <term>Base URI</term>
+              <term>Static base URI</term>
               <listitem>
                 <para>The base URI of the element on which the expression occurs.</para>
```
Comment (ndw) [inline langspec/xproc11/xproc.xml]: I don't think this is the static base URI; I think this is the "dynamic" base URI of the actual element in the document.

File: langspec/xproc11/xproc.xml [hunk 2]
```diff
@@ -1196,7 +1258,7 @@ readable port then the context node is undefined.</para>
...
```
Comment (ndw) [inline langspec/xproc11/xproc.xml]: Ditto here, re: base URI.

**Comment by ndw** [comment]
I made a couple of review comments. Aside from those, it looks fine to me.

File: langspec/xproc11/xproc.xml (hunk 1 above)
Comment (xml-project) [inline langspec/xproc11/xproc.xml]: I used the same term as in XPath 3.1 specs. They say "Static Base URI", but of course we are free to choose an other term.

File: langspec/xproc11/xproc.xml (hunk 2 above)
Comment (xml-project) [inline langspec/xproc11/xproc.xml]: I used the same term as in XPath 3.1 specs. They say "Static Base URI", but of course we are free to choose an other term.

**Comment by ndw** [comment]
Ok. I have some reservations about that definition, but I agree you've just brought it in line with 3.1.

**Pull request merged**